iceoryx_posh/test/moduletests/test_mepoo_chunk_header.cpp

```diff
@@ -16,6 +16,7 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Please adapt the copyright header.

Reply:
Everything is correct in there, Apex.AI is only in the first line. Woohoo

Reply:
Argh it has happened again :sweat_smile:  Can we please swap the lines? :rofl:

Reply:
Oh, right. Most probably there was a merge last year which shuffled this around

Reply:
@mossmaurice Declined! According to our contribution.md it is chronological and Apex was first here.